### PR TITLE
Removed search icon from faq list and admin controls

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

This PR simply removes the search icon from FAQListActivity and AdministratorControlsActivity. The search icon was not functional and therefore its correct to remove it for now and introduce it when we implement the whole functionality.

## Before vs. After

Notice top right corner.

  

  

## Checklist

- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ ] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/java/org/oppia/android/app/help/faq/FAQListActivity.kt [hunk 1]
```diff
@@ -21,11 +21,6 @@ class FAQListActivity : InjectableAppCompatActivity(), RouteToFAQSingleListener
     faqListActivityPresenter.handleOnCreate()
   }
 
-  override fun onCreateOptionsMenu(menu: Menu?): Boolean {
```
Comment (seanlip) [inline app/src/main/java/org/oppia/android/app/help/faq/FAQListActivity.kt:24]: Just to check, why is this removed?

Comment (rt4914) [reply app/src/main/java/org/oppia/android/app/help/faq/FAQListActivity.kt:24]: The search_icon was part of `menu_faq_list_activity` which is an independent xml file. 

This function `onCreateOptionsMenu ` helps to bind the `menu_faq_list_activity` to the Activity/Screen so that we can show the `search` icon in header/toolbar.

As we have removed the `search_icon`, it means that `menu_faq_list_activity` will be empty, therefore we remove it too which means we don't need to bind it to activity anymore.

**Review by seanlip — APPROVED** [review]
Ah ok, thanks! LGTM but let's get Veena's review before merging.

**Review by veena14cs — APPROVED** [review]
Thanks @rt4914 LGTM!
